pkg/server/registry/apps/strategy.go

```diff
@@ -90,10 +90,7 @@ func (s *Strategy) Validate(ctx context.Context, obj runtime.Object) (result fie
 }
 
 func (s *Strategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) (result field.ErrorList) {
```

Comment:
Not a big problem but I do notice that we no longer use the `old` parameter. Can we remove it or rename it to be `_`?

Reply:
We can't drop it because that would cause the function to not conform to the interface. Renaming to `_` would be ok, but im not going to wait for that. going to merge